setup.cfg

```diff
@@ -3,7 +3,7 @@ exclude = .eggs,ENV,build,docs/conf.py,venv
 
 [yala]
 linters=pylint,pycodestyle,isort
-pylint args = --disable=too-few-public-methods,too-many-instance-attributes,no-name-in-module,unnecessary-pass --ignored-modules=napps.amlight.coloring
+pylint args = --disable=too-few-public-methods,too-many-instance-attributes,no-name-in-module,unnecessary-pass,missing-timeout --ignored-modules=napps.amlight.coloring
```

Comment:
missing-timeout is still here, can you could check?

Reply:
Now it is delete.